Code/GraphMol/MolDraw2D/MolDraw2DSVG.h

```diff
@@ -41,32 +41,35 @@ class RDKIT_MOLDRAW2D_EXPORT MolDraw2DSVG : public MolDraw2D {
   // set font size in molecule coordinate units. That's probably Angstrom for
   // RDKit. It will turned into drawing units using scale_, which might be
   // changed as a result, to make sure things still appear in the window.
-  void setFontSize(double new_size);
-  void setColour(const DrawColour &col);
+  virtual void setFontSize(double new_size) override;
```

Comment:
Why is it desirable to mark all of these methods as `virtual`?
I don't think it's necessary to have methods of a derived class be virtual, and I don't think we'll be using this class as a base class for others.

Reply:
Maybe we could even mark the class as `final`

Reply:
One reason to use 

``` virtual f(int) override ```

is pragmatic.  If the type signature doesn't exist in the base class it is an compile error.  Otherwise you are making a new virtual function so the intent is a lot more clear.  This can help if the base class changes for some reason.